lib/telephony/otp_sender.rb

```diff
@@ -35,7 +35,6 @@ def send_confirmation_otp
 
     private
 
-    # rubocop:disable all
```

Comment:
There's a matching `# rubocop:enable all` below. Should it be removed?

Reply:
you'd think rubocop would catch those! 😛